app/presenters/menu/item.rb

```diff
@@ -1,5 +1,15 @@
 module Menu
   Item = Struct.new(:id, :name, :feature, :rbac_feature, :href, :type) do
+    extend ActiveModel::Naming
+
+    def self.base_class
+      Menu::Item
+    end
+
+    def self.base_model
+      model_name
+    end
+
```

Comment:
I don't think that creating a mixin with 6 lines is out of scope here, especially when you're introducing something new...